spec/02-integration/04-admin_api/02-apis_routes_spec.lua

```diff
@@ -20,7 +20,7 @@ pending("Admin API #" .. kong_config.database, function()
 
   setup(function()
     local _
-    _, db, dao = helpers.get_db_utils(kong_config.database)
+    _, db, dao = helpers.get_db_utils(kong_config.database, {})
```

Comment:
It seems like the second argument is optional, yet was added, empty, to many calls of `get_db_utils()` - for explicitness?